eng/pipelines/templates/variables/globals.yml

```diff
@@ -17,3 +17,7 @@ variables:
   DOTNET_NOLOGO: 1
   Package.EnableSBOMSigning: true
   Codeql.SkipTaskAutoInjection: true
```

Comment:
@benbp did you decide against having some properties similar to java that allow for setting the DefaultTestFilter and then just allow folks that want a custom TestFilter to create the full expression including the DefaultTestFilter (or perhaps excluding it if they don't want that)?

Reply:
Sorry for the wall of text but here is my current thinking on this:

I'm still worried about allowing an override of the default, as excluding or modifying it will change the nature of the pipeline, e.g. the only thing that really defines a live test pipeline is resource deployment and the test filter. If it's changeable in a text config, people can see what the default is and extend it. But since it's a variable it would also be accessible at pipeline runtime. I think we would see issues where people forgot to put the full statement in at runtime and/or do not know where to look to find the default, or we would have to set the variable default via pipeline generator somehow.

Also if this variable ends up getting used more widely, I don't want to have to refactor a ton of config files if we ever change the base filters for whatever reason. So for these reasons I think it's better to keep it "private."

We could consider having a powershell processing step to join the values like `(TestCategory==Live) & $(AdditionalTestFilters)` via the vso set variable command, that way users don't need to prefix their statement with a boolean operator? I guess currently it's still possible to ignore the inner test filter by using `|` instead of `&`.

I also tried a noop like `(TestCategory==Live) & ()` which sadly didn't work, but we could consider having the default variable value be something like `(TestCategory==Live) & (TestCategory!=PlaceholderValue)` that way we can hardcode the `&` into the inner value.

Reply:
Ok I can buy this argument. 

To solve the starts with `&` argument maybe just always have the `&` in the expression and default AddtionalTestFilters to some expression that is always true instead of empty.